schemas/v1/data_index.json

```diff
@@ -5,6 +5,9 @@
     "description": "Metadata and paths for Health-GPS input files",
     "type": "object",
     "properties": {
+        "$schema": {
+            "const": "https://raw.githubusercontent.com/imperialCHEPI/healthgps/main/schemas/v1/data_index.json"
+        },
         "version": { "const": 2 },
```

Comment:
Shall this be removed?

Reply:
Yes! Oops :see_no_evil:.

What's slightly concerning is that the file is still validating... I think this must mean "if the version field is present, it must be 2", but it doesn't enforce its presence. Ugh.

I've opened an issue for it: #403